advanced-cache.php

```diff
@@ -258,8 +258,14 @@ function ob($output) {
 		}
 
 		if ( !empty( $this->cache['headers'] ) && !empty( $this->uncached_headers ) ) {
-			foreach ( $this->uncached_headers as $header )
-				unset( $this->cache['headers'][$header] );
+			foreach ( $this->uncached_headers as $h ) {
```

Comment:
Any particular reason we can't use the full descriptive variable name? `$h` may be quicker to type but it hides the purpose of the variable and makes the rest of the code less readable